blockchain/chain_sync/src/chain_muxer.rs

```diff
@@ -899,19 +898,20 @@ mod tests {
         assert_eq!(root, expected_root);
     }
 
-    #[test]
-    fn empty_msg_meta_vector() {
-        let blockstore = MemoryDB::default();
-        let usm: Vec<UnsignedMessage> =
-            encoding::from_slice(&base64::decode("gA==").unwrap()).unwrap();
-        let sm: Vec<SignedMessage> =
-            encoding::from_slice(&base64::decode("gA==").unwrap()).unwrap();
-
-        assert_eq!(
-            TipsetValidator::compute_msg_root(&blockstore, &usm, &sm)
-                .expect("Computing message root should succeed")
-                .to_string(),
-            "bafy2bzacecmda75ovposbdateg7eyhwij65zklgyijgcjwynlklmqazpwlhba"
-        );
-    }
+    // FIXME: IPLD format changed. Figure out what this test was supposed to do.
```

Comment:
Let's have a task for that?

Reply:
Done. Tracking link added to the comment.